introduction.md

```diff
@@ -0,0 +1,3 @@
+My name is Frederico, I�m 42 years old guy from Brazil. I�m trying to improve my computer programing skills mastering GitHub.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.